File: opencv_webcam.cpp
#include <opencv2/opencv.hpp>

using namespace std;
using namespace cv;

int main(int ac, char** av) {

	VideoCapture cap(1);
	
	if (!cap.isOpened())
	{
		printf("Can't open the camera");
		return -1;
	}

	Mat img;

	while (1)
	{
		cap >> img;

		imshow("camera img", img);

		if (waitKey(1) == 27)
			break;
	}


	return 0;
}

File: opencv_webcam.py
import cv2

cap = cv2.VideoCapture(1)

if not cap.isOpened():
	raise IOError("Cannot open webcam")

while True:
	ret, frame = cap.read()
	if (ret):
		cv2.imshow('webcam', frame)
	if cv2.waitKey(1) & 0xFF == ord('q'):
		break
cap.release()
cv2.destroyAllWindows()